roles/kubernetes/master/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -269,7 +269,6 @@ controllerManager:
   extraArgs:
     node-monitor-grace-period: {{ kube_controller_node_monitor_grace_period }}
     node-monitor-period: {{ kube_controller_node_monitor_period }}
-    pod-eviction-timeout: {{ kube_controller_pod_eviction_timeout }}
```

Comment:
you can drop this commit (rebase on latest master)

Reply:
I moved this in an early commit, which is why the rebase didn't remove it. I've dropped it in that commit